test/factory/common_factory_tests.dart

```diff
@@ -1,3 +1,5 @@
+@Tags(const ["ddcFailure"])
```

Comment:
I think I misread the docs about this.. for some reason I thought I needed to declare the tag to be able to use it. I'll take this out.